zen/templates/__init__.py

```
@@ -0,0 +1,11 @@
+"""
+Template Pokédex package exposing registry-aware utilities.
+"""
+
+from .pokedex import TemplatePokedex  # noqa: F401
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)